1_Get_Started/practice/closure_dongbkim.js

```diff
@@ -0,0 +1,28 @@
+function range(start,end) {
+	if(end != undefined)
+		return makeArray(start,end);
+	return function endGetter(endNum){
+			return makeArray(start,endNum);
+		}
+
+	function makeArray(st,en){
+		let array = [];
+		for(let i = st;i <= en;i++)
```

Comment:
```suggestion
		for(let i = start;i <= en;i++)
```
클로저를 이용해 다음과 같이 수정한다면 함수의 매개변수를 줄일수 있을 것 같습니다.

Reply:
불필요한 매개변수가 있었군요.. 끝 숫자만 신경 써도 되니 하나 차이로 전체 코드가 깔끔해질 것 같습니다.  말씀 감사합니다!!